lammpslib.py

```diff
@@ -529,9 +552,11 @@ def propagate(self, atoms, properties, system_changes, n_steps, dt=None, dt_not_
         if not self.parameters.keep_alive:
             self.lmp.close()
 
-    def lammpsbc(self, pbc):
+    def lammpsbc(self, pbc, fix):
         if pbc:
             return 'p'
+        elif fix:
```

Comment:
Currently `fix` is always `False` [here](https://github.com/wexlergroup/pymatnest-surface/blob/92d042222236939c7e77479c0d6f56ec3e0fbff0/lammpslib.py#L662) but it can be better utilized in the future. Let's keep it.